components/datetime/tests/fixtures/tests/components.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "input": {
+            "locale": "en",
+            "value": "2020-01-21T08:25:07.000",
+            "options": {
+                "components": {
+                    "weekday": "long",
+                    "month": "long",
+                    "day": "two-digit",
+                    "year": "numeric",
+                    "hour": "numeric",
+                    "minute": "numeric",
+                    "second": "numeric",
+                    "time_zone_name": "long"
+                }
+            }
+        },
+        "output": {
+            "value": "Tuesday, January 21, 2020 at 8:25:07 AM zzzz"
```

Comment:
I only ported one of the style fixtures to components, but plan on porting me as a starting point. This will be in a future PR.